### PR TITLE
Add use casdoor as example to login dashboard by sso

### PR DESCRIPTION
**Pull request by Steve0x2a**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add use casdoor as example to login dashboard by sso

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7510
- Other reference link(s): https://github.com/casdoor/casdoor/issues/328

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-session-sso.md [hunk 1]
```diff
@@ -195,3 +195,42 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
...
+
+   ```
+   http://DASHBOARD_IP:PORT/dashboard/?sso_callback=1
+   ```
+   
+   在以上内容中，将 `DASHBOARD_IP:PORT` 替换为你在浏览器中实际访问 TiDB Dashboard 所使用的域名（或 IP）及端口。   
+
+   ![Settings](/media/dashboard/dashboard-session-sso-casdoor-settings-2.png)
+   
+6. 其他设置保持原样并点击**保存 & 退出**。
```
Comment (shichun-0415) [inline dashboard/dashboard-session-sso.md:224]:
```suggestion
6. 其他设置保留默认值，点击**保存 & 退出**。
```

File: dashboard/dashboard-session-sso.md [hunk 2]
```diff
@@ -195,3 +195,42 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
...
+
+7. 保存页面上的 **客户端 ID** 。
```
Comment (shichun-0415) [inline dashboard/dashboard-session-sso.md:226]:
```suggestion
7. 保存页面上的**客户端 ID**。
```

File: dashboard/dashboard-session-sso.md [hunk 3]
```diff
@@ -195,3 +195,42 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
...
+1. 部署并访问 Casdoor 的管理后台。
+
+2. 点击上方菜单栏的**应用**。
+
+3. 点击**添加**按钮。
+   ![Settings](/media/dashboard/dashboard-session-sso-casdoor-settings-1.png)
+
+4. 填写**名称**和**显示名称**，比如：**TiDB Dashboard** 。
+
+5. 在 **回调 URLs** 中添加如下内容：
```
Comment (shichun-0415) [inline dashboard/dashboard-session-sso.md:214]:
```suggestion
5. 在**回调 URLs** 中添加如下内容：
```

File: dashboard/dashboard-session-sso.md [hunk 4]
```diff
@@ -195,3 +195,42 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
...
+   
+6. 其他设置保持原样并点击**保存 & 退出**。
+
+7. 保存页面上的 **客户端 ID** 。
+
+### 步骤二：获取 TiDB Dashboard 所需的配置参数并填入 TiDB Dashboard
+
+1. 将上一步获得的**客户端 ID** 字段的值填入 TiDB Dashboard 的 **OIDC Client ID**。
+
+2. 将 **Casdoor** 部署地址加上 `https://` 前缀和 `/` 后缀后填入 **OIDC Discovery URL** 中，例如 `https://casdoor.example.com/`。完成授权并保存配置即可。
```
Comment (shichun-0415) [inline dashboard/dashboard-session-sso.md:232]:
```suggestion
2. 将 Casdoor 部署地址加上 `https://` 前缀和 `/` 后缀后填入 **OIDC Discovery URL** 中，例如 `https://casdoor.example.com/`。完成授权并保存配置即可。
```
In the English version, it is the **Domain** field, not the **Casdoor** 部署地址.

File: dashboard/dashboard-session-sso.md [hunk 5]
```diff
@@ -195,3 +195,42 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
...
+
+      ![Settings](/media/dashboard/dashboard-session-sso-casdoor-settings-3.png)
```
Comment (shichun-0415) [inline dashboard/dashboard-session-sso.md:234]:
```suggestion
        ![Settings](/media/dashboard/dashboard-session-sso-casdoor-settings-3.png)
```

**Review by breeswish — APPROVED** [review]
Thanks a lot!

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**